hashicorp/boundary-ui
# Icu 13897 create remove tag modal

# Description

Add a modal that removes a single worker tag




## Screenshots (if appropriate)
https://github.com/user-attachments/assets/801311a3-2ab6-45b2-98bb-7f85b242714a


<img width="510" alt="Screenshot 2024-07-09 at 5 35 20 PM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/21252a19-72de-47b2-b642-e041d4d3f876">


## How to Test

Navigate to the tags tab of a worker that has api tags. Click on the action at the end of that row and select remove tag. Type `REMOVE` to enable the Remove button. Tag should be removed from the list.

## Checklist
